modules/nf-scil/image/resample/main.nf

```diff
@@ -0,0 +1,53 @@
+process IMAGE_RESAMPLE {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://scil.usherbrooke.ca/containers/scilus_1.6.0.sif':
+        'scilus/scilus:1.6.0' }"
+
+    input:
+    tuple val(meta), path(image)
+
+    output:
+    tuple val(meta), path("*_resampled.nii.gz"), emit: image
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    def voxel_size = task.ext.voxel_size ? "--voxel_size " + task.ext.voxel_size : ""
+    def interp = task.ext.interp ? "--interp " + task.ext.interp : ""
+
+
+"""
+    export ITK_GLOBAL_DEFAULT_NUMBER_OF_THREADS=1
+    export OMP_NUM_THREADS=1
+    export OPENBLAS_NUM_THREADS=1
+    scil_resample_volume.py $image ${prefix}__resampled.nii.gz \
```

Comment:
Have you explored the additional options of the resampling script ? If I remember well, there are other ways to resample than the voxel size. Please, add them also.